<details><summary>Failed to submit graph for 2b0fd91e1e907bdb3fe4c36bb830bfa613852391
</summary>Message:
Failed to run command: 'pbp generate from-feedstock --output /tmp/tmpyonf_a0r/gdb-feedstock.yaml --custom-metadata-file /tmp/tmpyonf_a0r/custom_metadata.yaml --release-name gdb-feedstock-5-2b0fd91e1e907bdb3fe4c36bb830bfa613852391 gdb-feedstock/recipe', with error: 'INFO:root:Found feedstock gdb-feedstock at /tmp/tmpyonf_a0r/gdb-feedstock/recipe
INFO:root:Build/Test graph from gdb-feedstock
Created by sbx_user1051 on 2026-03-09 14:24:27
Command line: /opt/conda/envs/webhook-listener/bin/pbp generate from-feedstock --output /tmp/tmpyonf_a0r/gdb-feedstock.yaml --custom-metadata-file /tmp/tmpyonf_a0r/custom_metadata.yaml --release-name gdb-feedstock-5-2b0fd91e1e907bdb3fe4c36bb830bfa613852391 gdb-feedstock/recipe

INFO:root:Creating graph from feedstock at: gdb-feedstock
INFO:pbpcli.utils.common:Setting env var: CONDA_SUBDIR = linux-64.
INFO:pbpcli.utils.expand_variants:Elapsed time for get_package_variants: 0.11 seconds
INFO:pbpcli.utils.expand_variants:Matrix vars: 2 unchanging vars:(488), changing vars (2): ['blas_impl', 'python']
INFO:pbpcli.utils.expand_variants:Elapsed time for preparation for render: 0.00 seconds
INFO:pbpcli.utils.common:Setting env var: CONDA_PKGS_DIRS = /tmp/conda_pkgs_98aygt87.
WARNING: Number of parsed outputs does not match detected raw metadata blocks. Identified output block may be wrong! If you are using Jinja conditionals to include or exclude outputs, consider using `skip: true  # [condition]` instead.
/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py:540: UserWarning: The environment variable 'TMPDIR' specified in script_env is undefined.
  warnings.warn(
/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py:540: UserWarning: The environment variable 'TMPDIR' specified in script_env is undefined.
  warnings.warn(
/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/environ.py:540: UserWarning: The environment variable 'TMPDIR' specified in script_env is undefined.
  warnings.warn(
INFO:pbpcli.utils.common:Unsetting env var: CONDA_PKGS_DIRS
INFO:pbpcli.utils.common:Unsetting env var: CONDA_SUBDIR
Traceback (most recent call last):
  File "/opt/conda/envs/webhook-listener/bin/pbp", line 10, in <module>
    sys.exit(cli())
             ~~~^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1830, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 416, in wrapper
    return func(*args, **kwargs)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 643, in from_feedstock
    build_configs, test_configs = expand_build_matrix(
                                  ~~~~~~~~~~~~~~~~~~~^
        feedstock.recipe_dir_path, src_channels, parsed_user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/cli/generate.py", line 82, in expand_build_matrix
    _targets, build_configs, test_configs = variants_for_feedstock_and_platform(
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        recipe_dir_path, platform, src_channels, user_variants
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/pbpcli/utils/expand_variants.py", line 554, in variants_for_feedstock_and_platform
    metadata_tuples = api.render(
        parsed.recipe,
    ...<2 lines>...
        variants=user_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/api.py", line 68, in render
    return render_metadata_tuples(
        metadata_tuples,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 1097, in render_metadata_tuples
    for od, om in meta.get_output_metadata_set(
                  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        permit_undefined_jinja=not finalize,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        bypass_env_check=bypass_env_check,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    ):
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 2781, in get_output_metadata_set
    conda_packages = finalize_outputs_pass(
        ref_metadata,
    ...<3 lines>...
        bypass_env_check=bypass_env_check,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/metadata.py", line 1077, in finalize_outputs_pass
    fm = finalize_metadata(
        om,
        parent_metadata=parent_metadata,
        permit_unsatisfiable_variants=permit_unsatisfiable_variants,
    )
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 722, in finalize_metadata
    build_unsat, host_unsat = add_upstream_pins(
                              ~~~~~~~~~~~~~~~~~^
        m, permit_unsatisfiable_variants, exclude_pattern, extra_specs
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 507, in add_upstream_pins
    host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(
                                                       ~~~~~~~~~~~~~~~~~~~~~~~~^
        m, "host", permit_unsatisfiable_variants, exclude_pattern, extra_specs
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 463, in _read_upstream_pin_files
    extra_run_specs = get_upstream_pins(m, precs, env)
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 433, in get_upstream_pins
    loc, dist = execute_download_actions(
                ~~~~~~~~~~~~~~~~~~~~~~~~^
        m,
        ^^
    ...<2 lines>...
        package_subset=[prec],
        ^^^^^^^^^^^^^^^^^^^^^^
    )[prec]
    ^
  File "/opt/conda/envs/webhook-listener/lib/python3.13/site-packages/conda_build/render.py", line 388, in execute_download_actions
    link_prec = [
                ~
    ...<3 lines>...
        == (prec.name, prec.version, prec.build)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    ][0]
    ~^^^
IndexError: list index out of range
'
aws lambda request id: 77ccfbe4-02fc-5e9d-b225-99ccd8a41505
[View in Grafana](https://grafana.anacondaconnect.com/explore?schemaVersion=1&panes=%7B%2286g%22:%7B%22datasource%22:%22loki-central%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7B__aws_cloudwatch_log_group%3D%5C%22%2Faws%2Flambda%2Fpbp-graph-submitter-prod%5C%22%7D%20%7C%20json%20%7C%20requestId%20%3D%20%6077ccfbe4-02fc-5e9d-b225-99ccd8a41505%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki-central%22%7D,%22editorMode%22:%22builder%22,%22direction%22:%22backward%22%7D%5D,%22range%22:%7B%22from%22:%22now-3h%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22columns%22:%7B%220%22:%22Time%22,%221%22:%22Line%22%7D,%22visualisationType%22:%22table%22,%22labelFieldName%22:%22labels%22%7D%7D%7D%7D&orgId=1)</details>

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -73,11 +73,6 @@ $SRC_DIR/configure \
     --with-libiconv-prefix=$PREFIX \
     ${expat_flag:-} \
     || (cat config.log && exit 1)
-# Force all generated Makefiles to use MAKEINFO=true so submakes (bfd, etc.) don't run makeinfo.
-# Configure bakes MAKEINFO path into Makefiles; env/command-line don't propagate to all submakes.
-for f in $(find . -name Makefile); do
-  sed -i.bak 's/^MAKEINFO *=.*/MAKEINFO = true/' "$f" || true
-done
+
 make -j${CPU_COUNT} VERBOSE=1
 make install
-

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -60,10 +60,6 @@ export CXXFLAGS="${CXXFLAGS} -std=gnu++17"
 mkdir build
 cd build
 
-# Disable building .info docs to avoid Perl/texinfo ABI mismatch (e.g. pl526 vs system Perl).
-# Set before configure so generated Makefiles get it; pass to make so submakes inherit it.
-export MAKEINFO=true
-
 $SRC_DIR/configure \
     --prefix="$PREFIX" \
     --with-separate-debug-dir="$PREFIX/lib/debug:/usr/lib/debug" \
@@ -74,5 +70,5 @@ $SRC_DIR/configure \
     ${expat_flag:-} \
     || (cat config.log && exit 1)
 
-make -j${CPU_COUNT} VERBOSE=1
+make VERBOSE=2
 make install

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -31,6 +31,8 @@ requirements:
     - {{ cdt('libxcrypt-devel') }}  # [py<311]
   host:
     - python
+    # match texinfo's pl526 build (avoid makeinfo Perl XS ABI mismatch)
+    - perl =5.26.*
     - ncurses {{ ncurses }}
     - texinfo 6.5
     - xz {{ xz }}

